client/src/app/App.js

```diff
@@ -61,6 +61,7 @@ import History from './History';
 import { PluginsRoot } from './plugins';
 
 import css from './App.less';
+import './App-patch.css';
```

Comment:
Can we add a note here about why we are doing this? So it's easier for us to understand once we remove it. 

As far as I understand this includes

* Making sure element-templates styles are applied
* Sticky headers are disabled